tasks/files-to-granules/tests/data/payload.json

```diff
@@ -0,0 +1,24 @@
+{
+  "config": {
+    "granuleIdExtraction": "(MOD11A1\\..*)(\\.hdf|\\.cmr\\.xml|_[\\w]{1,}\\.jpg)",
+    "inputGranules": [
+      {
+        "granuleId": "MOD11A1.A2017200.h19v04.006.2017201090724",
+        "files": [
+          {
+            "name": "MOD11A1.A2017200.h19v04.006.2017201090724.hdf",
+            "bucket": "cumulus-internal",
+            "filename": "s3://cumulus-internal/file-staging/subdir/MOD11A1.A2017200.h19v04.006.2017201090724.hdf",
```

Comment:
To be consistent with previous tests, we should probably have fileTypes here, as we're expecting them to be used for workflows.

Reply:
👍ea44a15bc0